<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddC</class>
 <widget class="QDialog" name="AddC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add computer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="input_layout_name">
     <item>
      <widget class="QLabel" name="label_c_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_c_name">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_c_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_c_year">
     <property name="text">
      <string>Build year:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="input_c_year">
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="input_layout_type">
     <item>
      <widget class="QLabel" name="label_c_type">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_c_type">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="input_c_type">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Digital</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electronic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transistor</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="input_layout_build">
     <item>
      <widget class="QLabel" name="label_c_build">
       <property name="text">
        <string>Build status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_c_build">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="input_c_build">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Built</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not built</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_c_add">
     <property name="text">
      <string>Add computer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
